test/other/test_fflush_fs_exit.out

```diff
@@ -1,2 +1,2 @@
 Hey1
-Hey2
+Hey2
```

Comment:
Is this change needed?

Reply:
This is needed to accommodate `other.test_fflush_fs_exit` under `NODERAWFS` since that can flush streams without newlines.

Reply:
I see that makes sense.